src/Testing/InteractsWithMail.php

```diff
@@ -147,4 +147,9 @@ protected function seeMessage()
 
         return $this;
     }
+
+    protected function dontSeeMessage()
+    {
+        $this->assertTrue($this->getMessages()->isEmpty());
```

Comment:
I look at updating this with the first 2 suggestions when i get a minute, thanks.

I didn't make it fluent because I couldn't think of a scenario where this method would be chained with any of the other methods? All other methods are asserting something about a message, where here there would be no message, so you wouldn't want to chain.

Unless i'm missing something obvious?! 😕